src/script/view_model/CallingViewModel.ts

```diff
@@ -374,7 +374,25 @@ export class CallingViewModel {
         if (conversationEntity.isGroup() && !this.teamState.isConferenceCallingEnabled()) {
           this.showRestrictedConferenceCallingModal();
         } else {
-          await startCall(conversationEntity, CALL_TYPE.NORMAL);
+          const memberCount = conversationEntity.participating_user_ets().length;
+          if (memberCount > maxGroupSize) {
+            PrimaryModal.show(PrimaryModal.type.WITHOUT_TITLE, {
+              preventClose: true,
+              primaryAction: {
+                action: async () => await startCall(conversationEntity, CALL_TYPE.NORMAL),
+                text: t('groupCallModalPrimaryBtnName'),
+              },
+              secondaryAction: {
+                text: t('modalConfirmSecondary'),
+              },
+              text: {
+                message: t('groupCallConfirmationModalTitle', maxGroupSize),
```

Comment:
Shouldn't this be `memberCount` ?

```suggestion
                message: t('groupCallConfirmationModalTitle', memberCount),
```